dynamo_local/table.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Move this file to a new ./schemas/ dir as mentioned above

Reply:
Addressed in https://github.com/brave/go-sync/commit/7cf233613841990dc41f83072201b31c45f1af62